mempool/collector.go

```diff
@@ -127,9 +127,12 @@ func (t *MempoolDataCollector) mempoolTxns() ([]exptypes.MempoolTx, txhelpers.Me
 		//fee, _ := txhelpers.TxFeeRate(msgTx)
 		// log.Tracef("tx fee: GRM result = %f, msgTx = %f", tx.Fee, fee.ToCoin())
 
+		_, feeRate := txhelpers.TxFeeRate(msgTx)
+
 		txs = append(txs, exptypes.MempoolTx{
 			TxID:      msgTx.TxHash().String(),
 			Fees:      tx.Fee,
+			FeeRate:   feeRate.ToCoin(),
```

Comment:
Also need something similar in mempool/monitor, (`(*MempoolMonitor).TxHandler`).

Reply:
Looks like `(pgb *ChainDB).GetMempool` needs it too, but there seems to be a number of fields unset in that function too.  I don't recall a reason or see docs explaining why, so perhaps that's a bug.